tensorboard/webapp/metrics/views/main_view/main_view_test.ts

```diff
@@ -1612,10 +1612,10 @@ describe('metrics main view', () => {
         ]);
         const card2IndicatorAfter = fixture.debugElement.query(byCss.INDICATOR);
 
-        expect(card2IndicatorBefore.nativeElement).not.toBe(
+        expect(card2IndicatorBefore.nativeElement).not.toEqual(
           card2IndicatorAfter.nativeElement
         );
-        expect(card2IndicatorBefore.attributes['data-id']).not.toBe(
+        expect(card2IndicatorBefore.attributes['data-id']).not.toEqual(
```

Comment:
I think the first `not.toBe` was more equal since, while the deep values can be equal, I did not want two DOM nodes to be of the same instance. How often does this test fail?